connection.go

```diff
@@ -569,21 +605,23 @@ func (v *connection) initializeSSL(sslFlag string) error {
 	}
 
 	switch sslFlag {
-	case "server":
+	case tlsModeServer:
 		connectionLogger.Info("enabling SSL/TLS server mode")
 		v.conn = tls.Client(v.conn, &tls.Config{InsecureSkipVerify: true})
-	case "server-strict":
+	case tlsModeServerStrict:
 		connectionLogger.Info("enabling SSL/TLS server strict mode")
 		v.conn = tls.Client(v.conn, &tls.Config{ServerName: v.connURL.Hostname()})
 	default:
-		err := fmt.Errorf("unsupported tlsmode flag: %s - should be 'server', 'server-strict' or 'none'", sslFlag)
-		connectionLogger.Error(err.Error())
-		return err
+		connectionLogger.Info("enabling SSL/TLS custom mode")
```

Comment:
Adding a comment to indicate this is the implementation for "mutual ssl mode" will be helpful for other developers in the future.

Reply:
Done. Good suggestion, I agree that "custom TLS" doesn't have any hint that it is used for mutual ssl mode.